plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageLayoutHelper.java

```diff
@@ -168,12 +168,17 @@ public static String syncVolumeToVmDefaultFolder(DatacenterMO dcMo, String vmNam
     public static String syncVolumeToVmDefaultFolder(DatacenterMO dcMo, String vmName, DatastoreMO ds, String vmdkName, String excludeFolders) throws Exception {
 
         assert (ds != null);
+
         if (!ds.folderExists(String.format("[%s]", ds.getName()), vmName)) {
             s_logger.info("VM folder does not exist on target datastore, we will create one. vm: " + vmName + ", datastore: " + ds.getName());
-
             ds.makeDirectory(String.format("[%s] %s", ds.getName(), vmName), dcMo.getMor());
         }
 
+        if (!ds.folderExists(String.format("[%s]", ds.getName()), "fcd")) {
```

Comment:
use the constant "HypervisorHostHelper.VSPHERE_DATASTORE_BASE_FOLDER" for "_fcd_"

Reply:
sorry for the delay.. will update later this evening.